### ISSUE TITLE
Tabs Block Lack Accessibility

### ISSUE THREAD

**Issue by rizaardiyanto1412** [publishpress/PublishPress-Blocks#1427]
Hi There,

I just wanted to reach out and let you know that there are some accessibility issues with the Tabs Block. We use the Publish Press Tabs on a site that uses Userway, and we noticed that the Tabs functionality was not working when the Userway plugin was enabled.

This is what Userway mentioned as why the Tabs are inaccessible:

“It seems like the list of each tab counts on aria-labelledby attribute to work correctly. However, because the aria-labelledby isn’t connected correctly to an idof any element on the site, it’s a violation so we need to remove the attribute as part of accessibility remediations. I can see your team useshref instead of id for the label element; it’s not a correct technique and still a violation in terms of accessibility. (Please check the attached screenshot)

If your team is able to add the correct id to the label element, it will work correctly with our widget and be more accessibile.”

Original report: https://wordpress.org/support/topic/tabs-block-lack-accessibility/
